mail/views.py

```diff
@@ -237,3 +239,34 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class EmailBouncedView(APIView):
+    """
+    View class that handles HTTP requests to course team mail API
+    """
+    permission_classes = (MailGunWebHookPermission, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
+        """
+        POST method handler
+        """
+        event = request.POST.get("event", None)
+        recipient = request.POST.get("recipient", None)
+        error = request.POST.get("error", None)
+        message_headers = request.POST.get("message-headers", "NA")
+        log_error_on_bounce = request.POST.get("log_error_on_bounce", False)
+
+        if event == "bounced" and log_error_on_bounce:
+            error_msg = (
```

Comment:
Can you move this outside the if statement and use the text with both `log.error` and `log.debug`?